arguments.py

```diff
@@ -99,10 +103,44 @@ def initArgs():
 	parser.add_argument("-p", "--processes", type=int, default=-1,
 						help="Number of processes to use for rendering and export. Default: Number of processor cores (or hyperthreads, if supported)")
 
+
+	# Parse arguments once to get preset flag
+	args = parser.parse_args()
```

Comment:
Commandline arguments are parsed twice with this approach. While not a big overhead and I can't think of any other problems this would produce, it is a bit ugly. If anyone has a better idea for this, please share.

Reply:
We agreed that this still seems like the best approach for now